src/pages/Contact/Contact.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import styled from 'styled-components'
+
+
+const Row = styled.div`
+display: flex;
+`
+export default function Contact() {
+  return (
+    <div style={{padding: '0 120px'}}>
+      
+      <p style = {{fontSize: '2rem'}}>Contact Us</p>
+      <p> Lets us know what you think! In order to provide better service, please do not hesitate to give us your feedback. Thank you.</p>
+      <hr/>
+      <form style={{marginTop: '80px'}}>
```

Comment:
It would be great to use css class name attribute for the elements and then creating a stylesheet in order to make the styling global and maintainable. You can perform this later in the context of a refactoring task.